items.json

```diff
@@ -30930,9 +30930,9 @@
     "name": "Wooden Twohander",
     "culture": "Neutral",
     "type": "TwoHandedWeapon",
-    "price": 602,
-    "weight": 0.55,
-    "tier": 1.33279145,
+    "price": 636,
+    "weight": 0.52,

```

Comment:
these weapons are useless in current state. You should aim for at least tier 5 when it comes to low tier weapons